Configuration/StandardSequences/python/MagneticField_38T_PostLS1_cff.py

```diff
@@ -0,0 +1,13 @@
+### THIS CFF IS DEPRECATED!!!! ###
+# please use MagneticField_cff.py instead
```

Comment:
In the comment fields you suggest to replace this with `MagneticField_cff,` while in the command line you substitute it with `MagneticField_38T_cff:` please fix the one which is wrong.

Reply:
It really depends what users intended to get when they picked this cff.
The name suggests a fixed 3.8T map regardless of conditions, so MagneticField_38T_cff would be the verbatim replacement. This is why it is substituted with MagneticField_38T_cff in the command.
But most likely it has been picked because it looked like the right one at the time: I expect that in 99% of the cases the standard MagneticField_cff would be the right choice instead. Which is why users are recommended to use the latter instead. Usage of non-conditions-dependent configs should in fact be discouraged for general usage.

BTW (although it's probably clear) I am just reverting here these files as they were before,

Eventually going through all instances and fixing them one by one with the correct cff will be the right choice, see comment below.